jarjar/src/main/java/com/eed3si9n/jarjar/KeepProcessor.java

```diff
@@ -41,56 +41,57 @@ public boolean isEnabled() {
 
     public Set<String> getExcludes() {
         Set<String> closure = new HashSet<String>();
-        closureHelper(closure, roots);
+        recursiveProcessDependencies(closure, roots);
+
         Set<String> removable = new HashSet<String>(depend.keySet());
         removable.removeAll(closure);
         return removable;
     }
 
-    private void closureHelper(Set<String> closure, Collection<String> process) {
-        if (process == null)
+    private void recursiveProcessDependencies(Set<String> result, Collection<String> roots) {
```

Comment:
This method was just renamed to be more readable, let me know if you want me to undo this.